docs/4_4_1_can.adoc

```diff
@@ -772,6 +799,6 @@ A Network FMU communicates its current CAN node state via the <<StatusOpCodeCanL
 * The current CAN node state of a Network FMU shall be set to `ERROR_ACTIVE` if the value of REC < 128 and TEC < 128 and communicate via <<StatusOpCodeCanLowCut, Status operation>> to the Bus Simulation.
 * The current CAN node state of a Network FMU shall be set to `BUS_OFF` if the value of TEC > 255 and communicate via <<StatusOpCodeCanLowCut, Status operation>> to the Bus Simulation.
 
-If `org.fmi-standard.fmi-ls-bus.DirectConfirmation` (see <<direct-confirmation-parameter>>) is set to `true`, the <<ConfirmOpCodeCanLowCut, Confirm operation>> cannot be directly used as indicator to set the TEC value and will be incorrect under the rules outlined above.
-It should be considered to solve the error handling differently in this case or to disable it completely within the specified Network FMU.
-Also `org.fmi-standard.fmi-ls-bus.DirectConfirmation` shall be set to `true` and <<BusErrorOpCodeCanLowCut, Bus Error operations>> are not available in <<DirectCommunication, Direct Communication>> cases, so that the values for TEC and REC automatically remain zero in this case.
\ No newline at end of file
+If `org.fmi-standard.fmi-ls-bus.WaitForBusNotification` (see <<bus-notification-parameter>>) is set to `false`, the <<ConfirmOpCodeCanLowCut, Confirm operation>> cannot be directly used as indicator to set the TEC value and will be incorrect under the rules outlined above.
+Also <<BusErrorOpCodeCanLowCut, Bus Error operations>> are not available in this scenario, so that the values for TEC and REC automatically remain zero in this case.
+It should be considered to solve the error handling differently in this case or to disable it completely within the specified Network FMU.
```

Comment:
TEC/REC?

Reply:
We introduce TEC and REC at the beginning of this chapter and use the common CAN abbreviations TEC and REC for the rest. The chapter start with the following sentences: This chapter describes a possible implementation of the CAN error handling within Network FMUs using a rule set based on <<BusErrorOpCodeCanLowCut, Bus Error operations>>.
Each Network FMU will care for its own **Transmit Error Counter (TEC), Receive Error Counter (REC)** and current CAN node state.
The values for TEC and REC will be increased and decreased with respect to the `Error Code`, `Is Sender` and `Detected` arguments of a <<BusErrorOpCodeCanLowCut, Bus Error operation>> and are inherited from the original CAN error confinement rules.